src/Softeq.NetKit.Chat.SignalR/Hubs/Notifications/MessageNotificationService.cs

```diff
@@ -29,21 +29,29 @@ public class MessageNotificationService : BaseNotificationService, IMessageNotif
             _channelService = channelService;
         }
 
-        public async Task OnAddMessage(MessageResponse message, string callerConnectionId = null)
+        public async Task OnAddMessage(MessageResponse message, RecipientType recipientType, string callerConnectionId = null)
         {
             var clientIds = await GetChannelClientConnectionIdsAsync(message.ChannelId);
+            var exceptList = new List<string> {callerConnectionId};
+
+            if (recipientType == RecipientType.AllExceptMemberConnections)
```

Comment:
Maybe it would be better if we move this logic to separate method(s) which will return full exceptList list for enum?

Reply:
done